Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -704,11 +704,15 @@ public class TimeSlider: UIControl {
     }
     
     private var timer: Timer?
-    private static var KVOContext = 0
     private var map: AGSMap?
     private var scene: AGSScene?
     private var isObserving: Bool = false
+    private var reInitializeTimeProperties: Bool = false
     
+    private var mapLayersObservation : NSKeyValueObservation?
```

Comment:
This is fairly minor, but I would suggest being consistent in not putting a space (` `) between the property name and the colon (`:`).

Reply:
fixed.